mno_analysis_tools/compute_window_of_downtime.py

```diff
@@ -0,0 +1,91 @@
+import json
+import pytz
+import argparse
+from datetime import datetime
+
+from dateutil.parser import isoparse
+from temba_client.v2 import Message
+from core_data_modules.cleaners import PhoneCleaner
+from core_data_modules.logging import Logger
+
+
+log = Logger(__name__)
+log.set_project_name("ComputeWindowOfDowntime")
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Compute maximum window of time with 0 messages")
+    parser.add_argument("input_file_path", metavar="input file", 
+        help="File to read the raw data downloaded as json.",
+    )
+    parser.add_argument("output_file_path", metavar="output file",
+        help="File to write the raw data downloaded as json.",
+    )
+    parser.add_argument("operator", metavar="operator",
+        help="Operator that you'll need to analyze",
+    )
+    parser.add_argument("direction", metavar="direction of message",
+        help="Direction of the message either sent or received by the operator",
+    )
+    parser.add_argument("start_date", metavar="start date", type=lambda s: isoparse(s),
+        help="The start date as ISO 8601 string from which the window of downtime will be computed.",
+    )
+    parser.add_argument("end_date", metavar="end date", type=lambda s: isoparse(s),
+        help="The end date as ISO 8601 string to which the window of downtime computation will end",
+    )
+
+    args = parser.parse_args()
+
+    input_file_path = args.input_file_path
+    output_file_path = args.output_file_path
+    target_operator = args.operator 
+    target_direction = args.direction
+    start_date = args.start_date
+    end_date = args.end_date
+
+    with open(input_file_path, mode="r") as f:
+        output = json.loads(f.read())
+        messages = [Message.deserialize(val) for val in output]
+        log.info("Loading messages from file {file_name}...")
+        log.info(f"Loaded {len(messages)} messages")
+
+    msg_sent_on_timestamps = []
+    generated_outputs = []
```

Comment:
Please move this to L63. Declaring variables a long way from where they get used is confusing.

Also, please use a more descriptive name than "outputs"